integration_tests/suite/base/test_contexts.py

```diff
@@ -371,3 +375,50 @@ def test_bus_events(context):
     yield s.check_event, 'context_deleted', headers, confd.contexts(
         context['id']
     ).delete
+
+
+def test_create_contexts_parallel():
+    def create_context(responses, index):
+        response = confd.contexts.post(label='MyContext', wazo_tenant=DELETED_TENANT)
+        responses[index] = response
+
+    # create tenants (including DELETED_TENANT) in wazo-auth
+    BaseIntegrationTest.mock_auth.set_tenants(*ALL_TENANTS)
+
+    # check DELETED_TENANT does not exist in wazo-confd (not yet created)
+    response = confd.tenants(DELETED_TENANT).get(wazo_tenant=MAIN_TENANT)
+    response.assert_status(404)
+
+    threads = [None] * 5
+    responses = [None] * 5
+
+    for i in range(len(threads)):
+        threads[i] = Thread(target=create_context, args=(responses, i))
+        threads[i].start()
+
+    for i in range(len(threads)):
+        threads[i].join()
```

Comment:
How about a nicer interface like [ThreadPoolExecutor](https://www.digitalocean.com/community/tutorials/how-to-use-threadpoolexecutor-in-python-3#step-2-using-threadpoolexecutor-to-execute-a-function-in-threads)?